### PR TITLE
GAWB-1315 add PHI warning

### PR DESCRIPTION
**Pull request by jlmwhitt**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
`Warning.svg` did not render for me.  Do you need to add it to a resource folder?

Since the two divs are identical, please define it in one place only.

**Comment by jlmwhitt** [comment]
Added a component to common and use the component

File: src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs [hunk 1]
```diff
@@ -449,3 +449,10 @@
    (fn [{:keys [refs this]}]
      (.addEventListener (@refs "filter-field") "search" #(when (= (.-value (.-currentTarget %)) "") (react/call :apply-filter this))))})
 
+(react/defc PHIWarning
+  {:render
+   (fn [{:keys [props]}]
+     [:div {:style {:fontWeight "bold" :fontSize "106%" :marginBottom ".5em"}}
+       [:img {:src "assets/Warning.svg" :style {:verticalAlign "middle" :marginRight ".32em"}}]
+          "FireCloud is not intended to host personally identifiable information. Do not use any patient identifier, including name, social security number, or medical record number."])})
+
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs]: This doesn't need to be a React component, it should just be a div, probably defined in common.  Also check indentation.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs [hunk 2]
```diff
@@ -193,7 +193,10 @@
             (when (pos? count-all)
               [:ul {:style {:marginTop "0"}}
                (for [[status subs] (sort submissions-count)]
-                 [:li {} (str subs " " status)])])]))]]
+                 [:li {} (str subs " " status)])])]))
+       ]]
+    (when editing? [comps/PHIWarning])
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs]: Formatting.  The `]]` should go back on the previous line, and this isn't indented right.

File: src/static/assets/Warning.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><svg height="32px" version="1.1" viewBox="0 0 32 32" width="32px" xmlns="http://www.w3.org/2000/svg" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns" xmlns:xlink="http://www.w3.org/1999/xlink"><title/><defs/><g fill="none" fill-rule="evenodd" id="Icons new Arranged Names Color" stroke="none" stroke-width="1"><g fill="#FF0000" id="101 Warning"><path d="M14.4242327,6.14839275 C15.2942987,4.74072976 16.707028,4.74408442 17.5750205,6.14839275 L28.3601099,23.59738 C29.5216388,25.4765951 28.6755462,27 26.4714068,27 L5.5278464,27 C3.32321557,27 2.47386317,25.4826642 3.63914331,23.59738 Z M16,20 C16.5522847,20 17,19.5469637 17,19.0029699 L17,12.9970301 C17,12.4463856 16.5561352,12 16,12 C15.4477153,12 15,12.4530363 15,12.9970301 L15,19.0029699 C15,19.5536144 15.4438648,20 16,20 Z M16,24 C16.5522848,24 17,23.5522848 17,23 C17,22.4477152 16.5522848,22 16,22 C15.4477152,22 15,22.4477152 15,23 C15,23.5522848 15.4477152,24 16,24 Z M16,24" id="Triangle 29"/></g></g></svg>
```
Comment (MattPutnam) [inline src/static/assets/Warning.svg]: Can we just use the warning icon in icons.cljs?

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs [hunk 4]
```diff
@@ -29,6 +29,7 @@
                                (set! (.-onload reader)
                                      #(swap! state assoc :file file :file-contents (.-result reader)))
                                (.readAsText reader (.slice file 0 preview-limit)))))}]
+      [comps/PHIWarning]
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs]: Do we need this one here?  This causes it to be shown twice in the import data dialog.

**Review by MattPutnam — CHANGES_REQUESTED** [review]

**Comment by jlmwhitt** [comment]
Changed component to div from common, dropped the img and used the icon.

**Comment by jmthibault79** [comment]
👍

File: src/cljs/org/broadinstitute/firecloud_ui/common.cljs [hunk 5]
```diff
@@ -180,3 +182,10 @@
           (count (get-in entity ["attributes" (set-type->membership-attribute entity-type)]))
           ;; something nonsensical has been selected, submission will probably fail anyway:
           :else 1)))
+
+
+(def PHI-warning
+  [:div {:style {:fontWeight "bold" :fontSize "106%" :marginBottom ".5em"}}
+  (icons/icon {:style {:fontSize 48 :color (:exception-state style/colors) :marginRight ".15em" :verticalAlign "middle"}} :warning-triangle)
+    "FireCloud is not intended to host personally identifiable information. Do not use any patient identifier, including name, social security number, or medical record number."])
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/common.cljs:191]: Nit: indentation

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs [hunk 6]
```diff
@@ -37,6 +37,7 @@
          [:div {:style {:position "relative" :minWidth 500}}
           [:div {:style {:fontSize "150%" :marginBottom "1ex"}}
            [comps/Breadcrumbs {:crumbs (:crumbs @state)}]]
+          (do common/PHI-warning)
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs]: Don't need the `do`, just `common/PHI-warning`.  Same in the other place.

File: src/cljs/org/broadinstitute/firecloud_ui/common.cljs (hunk 5 above)
Comment (jlmwhitt) [reply src/cljs/org/broadinstitute/firecloud_ui/common.cljs:191]: Nit: document the style expected

**Pull request merged**